src/SFA.DAS.FindEpao.Infrastructure/Api/ApiClient.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+using SFA.DAS.FindEpao.Domain.Configuration;
+using SFA.DAS.FindEpao.Domain.Interfaces;
+
+namespace SFA.DAS.FindEpao.Infrastructure.Api
+{
+    public class ApiClient : IApiClient
+    {
+        private readonly HttpClient _httpClient;
+        private readonly FindEpaoApi _config;
+
+        public ApiClient (HttpClient httpClient, IOptions<FindEpaoApi> options)
+        {
+            _httpClient = httpClient;
+            _config = options.Value;
+        }
+        public async Task<TResponse> Get<TResponse>(IGetApiRequest request)
+        {
+            AddHeaders();
+
+            var response = await _httpClient.GetAsync(request.GetUrl).ConfigureAwait(false);
+
+            response.EnsureSuccessStatusCode();
```

Comment:
Maybe for a later story, but probably want to change this so that it can distinguish between an error and not found response. If not found then just return default